### PR TITLE
Home + Fonts

### PR DESCRIPTION
**Pull request by ahmed1hsn**
I've created home page and added fonts (Green Lantern and Raleway).

### CODE REVIEW

File: components/home/best-solution.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+const base = css`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-auto-rows: min-content;
+  padding: 1rem;
+  margin-bottom: 4rem;
+`
+
+const Post = () => (
```
Comment (SamroonMichael) [inline components/home/best-solution.js:16]: All our home page components have the same name variable which is **Post** please change each of them according to that component name as this is the best practice and would not cause any confusion.

File: components/home/best-solution.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                  padding: 1rem 0;`}>
+                Online exposure is the best promotion for your business at present. And Webanah is your superpower to improve your online presence. We work as a part of your business team to understand your goals and market and provide a complete package of services that would work as a booster in whole to take your business to a higher rank on search engines. We provide detailed services of web development, content marketing, SEO, branding and design, social media marketing, app development, and consulting. So, if you are looking to boost your business and website, Webanah is the best resort for your business with experienced designers and marketers on board.
+            </div>
+        </div>
+
+
+    </div>
+)
+
+export { Post }
```
Comment (SamroonMichael) [inline components/home/best-solution.js:57]: Can we use **export default** then the component name here

File: components/home/best-solution.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+/** @jsx jsx */
+
+import { css, jsx } from "@emotion/react";
```
Comment (SamroonMichael) [inline components/home/best-solution.js:3]: I would suggest to always keep the style import at last after import react, other components and libs

File: components/home/best-solution.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        css={base}
+    >
+        <div
+            css={css`
+              width: 400px;
+              margin-left: 10rem;
+              `}
+        >
+            <Image
+                src="/webana-assets/fantasized-ideas-turn-into-reality.png"
```
Comment (SamroonMichael) [inline components/home/best-solution.js:27]: The assets shouldn't be coming from the public/images folder?

File: components/home/home.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
+/** @jsx jsx */
```
Comment (SamroonMichael) [inline components/home/home.js:1]: Same comments as above

File: components/home/what-we-do.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
+/** @jsx jsx */
```
Comment (SamroonMichael) [inline components/home/what-we-do.js:1]: Same comments as above

File: components/home/why-we-stand-out.js [hunk 7]
```diff
@@ -0,0 +1,152 @@
+/** @jsx jsx */
```
Comment (SamroonMichael) [inline components/home/why-we-stand-out.js:1]: Same comments as above. But we can break this one more component like a card component which takes in the icon, heading, and text props. You can use this a parent component and have card as a child component in it.

File: components/layout.js [hunk 8]
```diff
@@ -0,0 +1,44 @@
+/** @jsx jsx */
```
Comment (SamroonMichael) [inline components/layout.js:1]: There is already a layout component in the folder structure which is a wapper over all the pages. I don't think you pull the master branch before making changes. This layout component is for what purpose?

File: pages/index.js [hunk 9]
```diff
@@ -1,11 +1,25 @@
+/** @jsx jsx */
+
+import Layout from "../components/layout"
+import * as home from "../components/home/home"
+import * as wwd from "../components/home/what-we-do"
+import * as bs from "../components/home/best-solution"
+import * as wwso from "../components/home/why-we-stand-out"
```
Comment (SamroonMichael) [inline pages/index.js:7]: This is okay but I would suggest using like this **import Home from '../components/home/home'** for all the import components. Of course the component names and path will be different I am just giving an example.

File: shared/sidebar.js [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+  height: 90vh;
+  background: #302d3d;
+`
+export const Item = styled.div`
+  color: white;
+  &:hover {
+    filter: drop-shadow(0 0 0.4rem white);
+  }
+`
+const MyLink = props => (
```
Comment (SamroonMichael) [inline shared/sidebar.js:33]: Where are we getting these props from?

**Review by SamroonMichael — CHANGES_REQUESTED** [review]
Overall good work fix those few things are run them see how it works

File: components/home/best-solution.js (hunk 4 above)
Comment (ahmed1hsn) [reply components/home/best-solution.js:27]: You are right. it's public/webana-assets/.

**Review by SamroonMichael — COMMENTED** [review]
Overall looks good but there are 3 things I need a bit of fix:
1) /** @jsx jsx */ and import {css, jsx} from "@emotion/react" please push it down after the other imports. Style imports should be separated then others.
2) Please use the Layout component and Head component on all pages.
3) Our images are still coming from /webana-assets/ like I mention before it should be public/images

**Comment by ahmed1hsn** [comment]
1. https://emotion.sh/docs/css-prop#jsx-pragma
2. Using them already.
3. public folder is default for assets, so /webana-assets/ is actually public/webana-assets/,  I changed the name /webana-assets/ to /images/ so new directory path is public/images/.

**Pull request merged**